01_data_collection.py

```diff
@@ -38,20 +42,21 @@
 # This typically uses 9 API calls
 prs = repo.get_pulls(state='closed')
 pr_ids = [pr.number for pr in prs]
+len_pr_ids = len(pr_ids)
+_LOGGER.info(f"There are {len_pr_ids} closed PR's in {ORG}/{REPO}")
 
 closed_prs_df = pd.DataFrame(pr_ids, columns=["closed_pr_ids"])
 CLOSED_PR_IDS_FILENAME = os.path.join(ORG + REPO + "CLOSED_PR_IDS.parquet")
 CLOSED_PRS_KEY = os.path.join(s3_input_data_path, ORG, REPO, "closed_prs")
 cc.upload_to_ceph(closed_prs_df, CLOSED_PRS_KEY, CLOSED_PR_IDS_FILENAME)
 
-for pr_id in pr_ids:
-    
-    # This typically uses 8 API calls per PR
-    d = process_pr.get_mi_parsed_pr(repo, pr_id, TOKEN, gh)
+for idx, pr in enumerate(prs):
+    _LOGGER.info(f"{idx+1}/{len_pr_ids}...PR's remaining")    
+    d = process_pr.parse_pr_with_mi(pr)
```

Comment:
Would also like to use the logging in the github handling notebook such as these for the remaining API calls https://github.com/redhat-et/time-to-merge-tool/blob/1c5774a65c10903f11aa62beae9ece0d4a3ed568/github_handling.py#L66

Can you use the function wrapper on `parse_pr_with_mi` like we were doing here https://github.com/redhat-et/time-to-merge-tool/blob/1c5774a65c10903f11aa62beae9ece0d4a3ed568/process_pr.py#L141

Reply:
Are you sure we want to add @github_handler to `parse_pr_with_mi` since it does not actually make any API calls to  github?

Reply:
> Would also like to use the logging in the github handling notebook such as these for the remaining API calls

I will add that in a subsequent PR

Reply:
When I checked, steps like pull_request.created_at, pull_request.as_issue() each seem to be making an API call to GitHub.

Reply:
ok added the github_handler :)

Reply:
thanks!